readme.md

```diff
@@ -381,64 +400,11 @@ var handleClickEvent = function(e){
     e.preventDefault();
   }
 }
-button.addEventListener("click", handleClickEvent);
-```
-
-## Timing Functions (10 minutes / 2:20)
```

Comment:
I would keep all this is but label it as extra material. In that way the students will still have the content should they need it later.